spec/models/form_profile_spec.rb

```diff
@@ -15,8 +15,7 @@
     described_class.instance_variable_set(:@mappings, nil)
     allow(Flipper).to receive(:enabled?).and_call_original
     allow(Flipper).to receive(:enabled?).with(:remove_pciu, anything).and_return(false)
-    allow(Flipper).to receive(:enabled?).with(:va_v3_contact_information_service, anything)
-                                        .and_return(false)
+    allow(Flipper).to receive(:enabled?).with(:disability_526_max_cfi_service_switch, anything).and_return(false)
   end
```

Comment:
This should have been added. This was added to the other form profile spec in master